### PR TITLE
fix(viewer): Update volume icon after UI is loaded

### PR DESCRIPTION
**Pull request by mxiao6**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @mxiao6 has signed the CLA. Thanks for the pull request!

File: src/lib/viewers/media/MediaBaseViewer.js [hunk 1]
```diff
@@ -209,6 +209,9 @@ class MediaBaseViewer extends BaseViewer {
         this.emit(VIEWER_EVENT.load);
 
         this.loadUI();
+        // If media is muted because of auto-play,
+        // the volume icon should be updated after the UI is loaded
+        this.updateVolumeIcon();
```
Comment (jstoffan) [inline src/lib/viewers/media/MediaBaseViewer.js]: Should this call live within `handleVolume`?

**Comment by jstoffan** [comment]
Does this work for Dash videos? It looks like `loadeddataHandler` is overridden in DashViewer.

File: src/lib/viewers/media/MediaBaseViewer.js (hunk 1 above)
Comment (mxiao6) [reply src/lib/viewers/media/MediaBaseViewer.js]: yeah, but `handleVolume` is called within `autoplay`, which happens before `loadUI`, so we have to call `updateVolumnIcon` explicitly again.

Comment (mxiao6) [reply src/lib/viewers/media/MediaBaseViewer.js]: Another way to do this is to move `autoplay` call after `loadUI`. What do you think?

**Comment by mxiao6** [comment]
> Does this work for Dash videos? It looks like `loadeddataHandler` is overridden in DashViewer.

DashViewer has no problem because `handleVolume` in `autoplay` happens after `loadUI` in `loadeddataHandler` in DashViewer.

File: src/lib/viewers/media/MediaBaseViewer.js (hunk 1 above)
Comment (jstoffan) [reply src/lib/viewers/media/MediaBaseViewer.js]: Sure It seems like DashViewer and MediaBaseViewer should be as similar as possible in this regard.

**Pull request merged**